backend/src/middleware/hashtags/hashtagsMiddleware.spec.js

```diff
@@ -129,10 +148,29 @@ describe('hashtags', () => {
         )
       })
 
-      describe('afterwards update the Post by removing a Hashtag, leaving a Hashtag and add a Hashtag', () => {
-        // The already existing Hashtag has no class at this point.
-        const postContent =
-          '<p>Hey Dude, <a class="hashtag" href="/search/hashtag/Elections">#Elections</a> should work equal for everybody!? That seems to be the only way to have equal <a href="/search/hashtag/Liberty">#Liberty</a> for everyone.</p>'
+      describe('updates the Post by removing, keeping and adding one hashtag respectively', () => {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-17T07:30:01Z" title="Tuesday, September 17th 2019, 9:30:01 am +02:00">Sep 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -129,10 +148,29 @@ describe('hashtags', () => {
         )
       })
 
-      describe('afterwards update the Post by removing a Hashtag, leaving a Hashtag and add a Hashtag', () => {
-        // The already existing Hashtag has no class at this point.
-        const postContent =
-          '<p>Hey Dude, <a class="hashtag" href="/search/hashtag/Elections">#Elections</a> should work equal for everybody!? That seems to be the only way to have equal <a href="/search/hashtag/Liberty">#Liberty</a> for everyone.</p>'
+      describe('updates the Post by removing, keeping and adding one hashtag respectively', () => {
```


This is a false successful test !!! 🤔
Because the `data-hashtag-id` doesn’t make it in the editor again.
On reedit it is no hashtag link anymore.
Therefore here you create a test that’s not existent in reality.
That’s why we used the `href` to search for a hashtag.